cmd/kube-apiserver/app/aggregator.go

```diff
@@ -175,6 +175,7 @@ var apiVersionPriorities = map[schema.GroupVersion]priority{
 	// to my knowledge, nothing below here collides
 	{Group: "apps", Version: "v1beta1"}:                          {group: 17800, version: 1},
 	{Group: "apps", Version: "v1beta2"}:                          {group: 17800, version: 1},
+	{Group: "events.k8s.io", Version: "v1beta1"}:                 {group: 17750, version: 5},
```

Comment:
beta vs alpha?

Reply:
Yup - it has to be beta, so that components can actually use it.

Reply:
@gmarek Why does it have to be beta? Start with alpha and a feature gate like everything else?